lab3/src/pages/dashboard/calendar/index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Main Page</title>
+    <link rel="stylesheet" href="/styles/index.css" />
+  </head>
+  <body>
+    <div class="dashboard">
+      <div class="sidebar" id="sidebar">
```

Comment:
There's a better tag fro this.

Reply:
ok ,but i found another way, i guess ,its better than tag aside
https://aastudio.fr/Sidebar-and-Aside-are-different.html#:~:text=is%20an%20HTML5%20tag,of%20the%20content%20they%20carry

Reply:
fixed on aside